.github/workflows/test.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    types: [opened, synchronize, reopened, labeled]
```

Comment:
Not sure about the reason for the "labeled" part; but if labels are affecting the behavior of the tests, maybe we should also include "unlabeled" (removing a label)?

(Maybe it is sufficient as is, since it only cares about the presence of the "package" tag. But just asking to clarify.)